WordPress/Classes/ViewRelated/Blog/SiteManagement/TableViewHeaderDetailView.swift

```diff
@@ -112,35 +92,21 @@ open class TableViewHeaderDetailView: UITableViewHeaderFooterView {
     }
 
     fileprivate func stackSubviews() {
+
         contentView.addSubview(stackView)
         stackView.addArrangedSubview(titleLabel)
         stackView.addArrangedSubview(detailLabel)
 
-        stackView.centerXAnchor.constraint(equalTo: contentView.centerXAnchor).isActive = true
-        stackWidthConstraint = stackView.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width)
-        stackWidthConstraint?.isActive = true
+        NSLayoutConstraint.activate([
+            stackView.leadingAnchor.constraint(equalTo: contentView.readableContentGuide.leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: contentView.readableContentGuide.trailingAnchor),
+            stackView.topAnchor.constraint(equalTo: contentView.readableContentGuide.topAnchor),
```

Comment:
Since this is only a single `UILabel` in the layout, with text content, we technically want to use the `readableContentGuide` for top and bottom as well. (Even though it ends up being identical to using top and bottom from `layoutMarginsGuide`) A bit more here: https://developer.apple.com/reference/uikit/uiview/1622644-readablecontentguide